app/components/ShareButton.tsx

```diff
@@ -0,0 +1,32 @@
+'use client';
```

Comment:
Possibly make this component like the heart button, then can put on explore page also